language_server/src/lsp/feature_workspace_folders.cpp

```diff
@@ -173,6 +179,41 @@ void feature_workspace_folders::send_configuration_request()
         });
 }
 
+void feature_workspace_folders::register_file_change_notifictions()
+{
+    static const nlohmann::json global_patterns {
+        {
+            "registrations",
+            nlohmann::json::array_t {
+                {
+                    { "id", "global_watchers" },
+                    { "method", "workspace/didChangeWatchedFiles" },
+                    {
+                        "registerOptions",
+                        {
+                            {
+                                "watchers",
+                                nlohmann::json::array_t {
+                                    { { "globPattern", "**/*" } },
+                                    { { "globPattern", ".hlasmplugin/*.json" } },
```

Comment:
Just to verify - this is purely because we want to watch also "hidden" directories and this is a way to convince VS Code to do so, right?

Reply:
On linux (and macos I think), the hidden directories seem to be excluded from the "**/*" pattern unless explicitly mentioned.